packages/gestalt/src/DropdownItem.js

```diff
@@ -46,7 +46,6 @@ export default function DropdownItem({
           id={id}
           index={index}
           key={`${option.value + index}`}
-          lineClamp={1}
```

Comment:
Are these removals intentional? What purpose do they serve? If we do need to change Dropdown / OptionItem, can that happen in a separate PR, or is it related to this ComboBox work?

Reply:
Oh I see, lineClamp was only needed for ComboBox?